enterprise/frontend/src/embedding-sdk/docs/README.md

```diff
@@ -0,0 +1,62 @@
+# Metabase embedding SDK for React
+
+Metabase embedding SDK for React offers a way to integrate Metabase into your application in a more seamless way compared to the current interactive embedding offering.
+
+## Installation
+
+You can install Metabase embedding SDK for react via npm:
+
+```bash
+npm install @metabase/embedding-sdk-react
+```
+
+or using yarn:
+
+```bash
+yarn add @metabase/embedding-sdk-react
+```
+
+## Usage
+
+Once installed, you can need to import `MetabaseProvider` and provide it a `config` object.
+
+```jsx
+import React from "react";
+import { MetabaseProvider, useQuestionSearch, InteractiveQuestion} from "@metabase/embedding-sdk-react";
+
+// JWT configuration
+const jwtConfig = {
+  metabaseInstanceUrl: "https://metabase.example.com", // Required: Your Metabase instance URL
+  authType: "jwt", // Required: "jwt" or "apiKey"
+  jwtProviderUri: "https://app.example.com/sso/metabase", // Required: Your endpoint that returns JWT token used to authenticate Metabase
+  font: "Lato", // Optional: you could provide any fonts support by Metabase
+}
+
+// API Key configuration
+const apiKeyConfig = {
+  metabaseInstanceUrl: "https://metabase.example.com",
+  authType: "apiKey",
+  apiKey: "your_metabase_api_key" // Required: API Key created in your Metabase admin settings.
+}
+
+export default function App() {
+  const { data: [question], isLoading } = useQuestionSearch();
```

Comment:
Maybe add a small comment that this returns the first question. I was a bit confused at first as the hook ends in `Search`, so I wasn't aware that this hook just returns all questions if the query term is not provided.

Reply:
fair enough. Actually I even we provide a search team it would still return an array since it's calling `useEntityListQuery` under the hood
https://github.com/metabase/metabase/blob/4463c31cc02781c344ae52cc81a5cc2702264caf/enterprise/frontend/src/embedding-sdk/hooks/public/use-question-search.ts#L21-L24


https://github.com/metabase/metabase/blob/4463c31cc02781c344ae52cc81a5cc2702264caf/frontend/src/metabase/common/hooks/entity-framework/use-search-list-query/use-search-list-query.ts#L23-L30